### ISSUE TITLE
Can't download torrent if it has an apostrophe in its name

### ISSUE THREAD

**Issue by Daniel15** [piejanssens/premiumizer#339]
If a torrent has an apostrophe in its name, the call to the `category_selected`  function does not properly escape it.

**Comment by piejanssens** [comment]
@Daniel15 Please test the latest version and confirm. Thanks.

**Comment by wilki24** [comment]
Fixed it for me. However, I could not get the update to work. It just kept restarting, without any update. I did a clean reinstall, and everything works as expected, so I suspect it was just something related to my setup, but I mention it here just in case others have a similar issue.

**Comment by Daniel15** [comment]
I'll give it a test when I get a chance in the next few days. Does this also fix the category selector? I had issues not just with the download but also with the category selector when it says "Waiting on category"

**Comment by wilki24** [comment]
Yes, you can now choose a category and the download starts as expected.
